src/Containers/Microsoft.NET.Build.Containers/Resources/Strings.resx

```diff
@@ -358,26 +355,54 @@
     <comment/>
   </data>
+  <data name="ContainerBuilder_StartBuildingImageForRid" xml:space="preserve">
+    <value>Building image '{0}' for runtime identifier '{1}' on top of base image '{2}'.</value>
+    <comment/>
+  </data>
   <data name="BuildingImageIndex" xml:space="preserve">
     <value>Building image index '{0}' on top of manifests {1}.</value>
     <comment>
       {0} is the name of the image index and its tag, {1} is the list of manifests digests
     </comment>
   </data>
   <data name="MixedMediaTypes" xml:space="preserve">
-    <value>'mediaType' of manifests should be the same in manifest list (image index).</value>
+    <value>'mediaType' of manifests should be the same in image index.</value>
     <comment/>
   </data>
   <data name="UnsupportedMediaType" xml:space="preserve">
-    <value>Cannot create manifest list (image index) for the provided 'mediaType' = '{0}'.</value>
+    <value>Cannot create image index for the provided 'mediaType' = '{0}'.</value>
     <comment/>
   </data>
   <data name="ImagesEmpty" xml:space="preserve">
-    <value>Cannot create manifest list (image index) because no images were provided.</value>
+    <value>Cannot create image index because no images were provided.</value>
     <comment/>
   </data>
   <data name="InvalidImageMetadata" xml:space="preserve">
-    <value>Cannot create manifest list (image index) because provided images are invalid. Items must have 'Config', 'Manifest', 'ManifestMediaType' and 'ManifestDigest' metadata.</value>
+    <value>Cannot create image index because provided images are invalid. Items must have 'Config', 'Manifest', 'ManifestMediaType' and 'ManifestDigest' metadata.</value>
+    <comment/>
+  </data>
+  <data name="InvalidImageManifest" xml:space="preserve">
+    <value>Cannot create image index because at least one of the provided images' manifest is invalid.</value>
+    <comment/>
+  </data>
+  <data name="InvalidImageConfig" xml:space="preserve">
+    <value>Cannot create image index because at least one of the provided images' config is invalid.</value>
+    <comment/>
+  </data>
+  <data name="ImageConfigMissingArchitecture" xml:space="preserve">
+    <value>Cannot create image index because at least one of the provided images' config is missing 'architecture'.</value>
+    <comment/>
+  </data>
+  <data name="ImageConfigMissingOs" xml:space="preserve">
+    <value>Cannot create image index because at least one of the provided images' config is missing 'os'.</value>
+    <comment/>
+  </data>
+  <data name="ImageIndex_PodmanNotSupported" xml:space="preserve">
+    <value>Image index creation for Podman is not supported.</value>
+    <comment/>
+  </data>
+  <data name="TipToEnableContainerdForMultiArch" xml:space="preserve">
+    <value>Tip: For multi-arch image publishing, ensude that 'Use containerd for pulling and storing images' is checked in Docker Desktop settings.</value>
```

Comment:
```suggestion
    <value>Tip: For multi-arch image publishing, ensure that 'Use containerd for pulling and storing images' is checked in Docker Desktop settings.</value>
```

Reply:
thanks! i later decided to change this. instead of guessing that the containrd image store is disabled based on the error from docker, i now check if it is enabled before loading the image